src/main/java/com/google/cloud/spanner/hibernate/SpannerTableExporter.java

```diff
@@ -89,10 +89,18 @@ public SpannerTableExporter(SpannerDialect spannerDialect) {
             + " " + col.getSqlType(this.spannerDialect, metadata)
             + (col.isNullable() ? this.spannerDialect.getNullColumnString() : " not null")));
 
-    return new String[]{
-        MessageFormat.format(this.createTableTemplate, table.getQuotedName(),
+    ArrayList<String> statements = new ArrayList<>();
+    statements.add(MessageFormat.format(this.createTableTemplate, table.getQuotedName(),
             colsAndTypes.toString(),
-            primaryKeyColNames)};
+        primaryKeyColNames));
```

Comment:
nit: indentation of `primaryKeyColNames` does not seem quite right.

Reply:
Fixed.